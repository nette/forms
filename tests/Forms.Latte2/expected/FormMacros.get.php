<?php
%A%
		echo Nette\Bridges\FormsLatte\Runtime2::renderFormBegin($form = $this->global->formsStack[] = $this->global->uiControl["myForm"], []) /* line 1 */;
		echo Nette\Bridges\FormsLatte\Runtime2::renderFormEnd(array_pop($this->global->formsStack));
		echo '

';
		$form = $this->global->formsStack[] = $this->global->uiControl["myForm"] /* line 3 */;
		echo '<form';
		echo Nette\Bridges\FormsLatte\Runtime2::renderFormBegin(end($this->global->formsStack), [], false);
		echo '>
';
		echo Nette\Bridges\FormsLatte\Runtime2::renderFormEnd(array_pop($this->global->formsStack), false) /* line 3 */;
		echo '</form>
';
%A%
