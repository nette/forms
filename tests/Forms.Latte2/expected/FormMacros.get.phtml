<?php
%A%
		echo Nette\Bridges\FormsLatte\Runtime::renderFormBegin($form = $this->global->formsStack[] = $this->global->uiControl["myForm"], []) /* line 1 */;
		echo Nette\Bridges\FormsLatte\Runtime::renderFormEnd(array_pop($this->global->formsStack));
		echo '

';
		$form = $this->global->formsStack[] = $this->global->uiControl["myForm"] /* line 3 */;
		echo '<form';
		echo Nette\Bridges\FormsLatte\Runtime::renderFormBegin(end($this->global->formsStack), [], false);
		echo '>
';
		echo Nette\Bridges\FormsLatte\Runtime::renderFormEnd(array_pop($this->global->formsStack), false) /* line 3 */;
		echo '</form>
';
%A%
