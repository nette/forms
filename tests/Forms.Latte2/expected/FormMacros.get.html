<form action="" method="get"><input type="hidden" name="arg" value="val"><!--[if IE]><input type=IEbug disabled style="display:none"><![endif]-->
</form>


<form action="" method="get">
<input type="hidden" name="arg" value="val"><!--[if IE]><input type=IEbug disabled style="display:none"><![endif]-->
</form>
